share/tools/web_config/sample_prompts/starfish.fish

```diff
@@ -0,0 +1,34 @@
+# vim: filetype=sh
+# name: Starfish
+# author: Radu Potop
+# description: A prompt theme similar to Starship.rs but without the emojis.
+
+function fish_prompt --description 'Write out the prompt'
+    set -l last_status $status
+    set -l normal (set_color normal)
+    set -l status_color (set_color brgreen)
+    set -l cwd_color (set_color $fish_color_cwd)
+    set -l vcs_color (set_color brpurple)
+    set -l prompt_status ""
+
+    # Since we display the prompt on a new line allow the directory names to be longer.
+    set -g fish_prompt_pwd_dir_length 10
```

Comment:
Should this be `set -g`? Do we want to set it outside the prompt or should this be `set -lx`?

Reply:
It's fine to have these well-known variables as global.

You typically *don't* want to set these outside of the prompt because that makes `funced` annoying (and because fish_config basically just does `funcsave fish_prompt; funcsave fish_right_prompt; funcsave fish_mode_prompt` when picking a new prompt).

Reply:
Fixed the variable setting, thanks for the suggestion